packages/pwa-kit-runtime/src/ssr/server/express.js

```diff
@@ -460,3 +463,102 @@ export const getRuntime = () => {
 
     return boundRuntime
 }
+
+/**
+ * Patches `res.setHeader` to ensure that the Content-Security-Policy header always includes the
+ * directives required for PWA Kit to work.
+ * @param {express.Request} req Express request object
+ * @param {express.Response} res Express response object
+ * @param {express.NextFunction} next Express next callback
+ */
+export const defaultSecurityHeaders = (req, res, next) => {
```

Comment:
@wjhsf would you consider putting this code in `packages/pwa-kit-runtime/src/utils`? We could always re-export from this file to maintain the interface you describe in the changelog.